package.json

```diff
@@ -26,15 +26,15 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "eslint-plugin-unused-imports": "^2.0.0",
     "ioredis": "^4.28.2",
-    "jest": "^27.4.2",
+    "jest": "^27.5.1",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.5.1",
-    "typescript": "^4.5.2"
+    "typescript": "^4.6.2"
   },
   "scripts": {
-    "check": "run-p check:* --aggregate-output -l",
+    "check": "run-p check:* --aggregate-output --continue-on-error",
     "check:type": "npm run check:type --workspaces --if-present",
-    "check:lint": "eslint \"packages/**/*.[tj]s?(x)\" --cache --cache-strategy content",
+    "check:lint": "eslint \"packages/**/*.[tj]s?(x)\" --cache --cache-strategy content --max-warnings=0",
```

Comment:
lint warnings will now fail CI

Reply:
Over the years, I have come to regard warnings with suspision.
Either we care about something being bad in our code, in which case we should error.
Or we don't care, in which case there is no point in warning.
The worst scenario is that we care but only warn, which almost always goes undetected.
So 💯